src/main/java/classify/commands/ListStudentsCommand.java

```diff
@@ -0,0 +1,111 @@
+package classify.commands;
```

Comment:
Good use of OOP and SOC in making a new class!
![image](https://github.com/AY2324S2-CS2113-T13-3/tp/assets/102575848/eb437322-cd07-4b07-a21f-c9f05a1c37a4)